test/e2e/specs/editor/various/block-deletion.spec.js

```diff
@@ -287,15 +287,16 @@ test.describe( 'Block deletion', () => {
 		await expect.poll( editor.getBlocks ).toMatchObject( [
 			{ name: 'core/paragraph', attributes: { content: 'First' } },
 			{ name: 'core/paragraph', attributes: { content: 'Second' } },
-			{ name: 'core/paragraph', attributes: { content: '' } },
 		] );
 
 		// Ensure that the newly created empty block is focused.
-		await expect.poll( editor.getBlocks ).toHaveLength( 3 );
+		await expect.poll( editor.getBlocks ).toHaveLength( 2 );
```

Comment:
Why you had to change this?

Reply:
Because it's the only thing in the e2e tests that this PR changed in behaviour. Arrow up selects two blocks. Then press Backspace. Instead of leaving an empty block, it no longer leaves an empty block and puts selection at the end of the previous block.